src/LaunchDarkly.EventSource/Configuration.cs

```diff
@@ -10,15 +10,15 @@ namespace LaunchDarkly.EventSource
     /// A class used 
     /// </summary>
     public sealed class Configuration
-    {
+    {
```

Comment:
There are some spurious changes like this in the diff, which I believe were due to the file having previously had inconsistent linefeeds from being edited on Mac vs. Windows. The linefeeds are now consistent, and you can see the PR more clearly by selecting "Hide whitespace changes" in GitHub.

As part of the 4.0 update I'll be fixing any other linefeed issues like this, and then using `.gitattributes` to ensure that linefeeds are translated correctly in future commits.